##Assignment3.1

#Task 1.1: Write a function to compute 5/0 and use try/except to catch the exceptions.

def cal_div(div=0,d=0):
    res=div/d
    return res
try:
    q= cal_div(5) # function to compute 5/0
except ZeroDivisionError:
    print("ZerodivisonError:You are trying to do illegal divide by 0")
except:
    print("This is generic excception")
    
